style.css

```diff
@@ -381,6 +395,122 @@ hr {
   box-shadow: none;
 }
 
+#popup-project {
+  display: none;
+}
+
+#popup-project.active {
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  align-self: center;
+  justify-self: center;
+  height: 95%;
```

Comment:
Good job on adding technologies as the previous reviewer suggested. Unfortunately, now, the buttons are not fitting in the pop-up anymore.
Your design:
![image](https://user-images.githubusercontent.com/83596135/167898217-2a41daee-3345-4a4f-aa41-fda10d7a3541.png)

- [ ] Kindly refactor so that the buttons and technologies fit in the pop-up (you can refactor height of the container even if this will give a vertical scroll);

Reply:
Done that! Thank you.